docs/data-sources/resource.md

```diff
@@ -0,0 +1,61 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_resource Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Resources in Twingate represent any network destination address that you wish to provide private access to for users authorized via the Twingate Client application. Resources can be defined by either IP or DNS address, and all private DNS addresses will be automatically resolved with no client configuration changes. For more information, see the Twingate documentation https://docs.twingate.com/docs/resources-and-access-nodes.
+---
+
+# twingate_resource (Data Source)
+
+Resources in Twingate represent any network destination address that you wish to provide private access to for users authorized via the Twingate Client application. Resources can be defined by either IP or DNS address, and all private DNS addresses will be automatically resolved with no client configuration changes. For more information, see the Twingate [documentation](https://docs.twingate.com/docs/resources-and-access-nodes).
+
+## Example Usage
+
+```terraform
+data "twingate_resource" "foo" {
+  id = "<your resource's id>"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `id` (String) The ID of the Resource. The ID for the Resource must be obtained from the Admin API.
+
+### Read-Only
+
+- `address` (String) The Resource's address, which may be an IP address, CIDR range, or DNS address
+- `name` (String) The name of the Resource
+- `protocols` (List of Object) By default (when this argument is not defined) no restriction is applied, and all protocols and ports are allowed. See below for [nested schema](#nestedatt--protocols). (see [below for nested schema](#nestedatt--protocols))
```

Comment:
There's duplicate text at the end of this sentence. The final version should look like this:

`By default (when this argument is not defined) no restriction is applied, and all protocols and ports are allowed. See below for [nested schema](#nestedatt--protocols).`

Reply:
there is no config option to disable it

https://github.com/hashicorp/terraform-plugin-docs/blob/main/schemamd/render.go#L89

Reply:
Ah yeah, I realized this afterwards